docs/components/modeler/reference/modeling-guidance/rules/history-time-to-live.md

```diff
@@ -8,18 +8,20 @@ import MarkerGuideline from "@site/src/mdx/MarkerGuideline";
 
 <span class="badge badge--platform">Camunda 7 only</span>
 
-Each execution of a model resource (BPMN, DMN, and CMMN) generates historic data during execution that is stored in the database. In Camunda 7 [history cleanup](https://docs.camunda.org/manual/latest/user-guide/process-engine/history/) removes this historic data from the database based on a defined history time to live (HTTL).
+Each execution of a model resource (BPMN, DMN, and CMMN) generates historic data during execution that is stored in the database. In Camunda 7, [history cleanup](https://docs.camunda.org/manual/latest/user-guide/process-engine/history/) removes this historic data from the database based on a defined **history time to live (HTTL)**.
 
-Starting [from Camunda 7.20](https://docs.camunda.org/manual/develop/update/minor/719-to-720/#enforce-history-time-to-live) you must configure HTTL in one of the following ways:
+From [Camunda 7.20](https://docs.camunda.org/manual/develop/update/minor/719-to-720/#enforce-history-time-to-live) onwards, you must configure HTTL in one of the following ways:
 
-- Define HTTL per model directly in the Camunda Desktop Modeler.
+- Define HTTL per model directly in Desktop Modeler.
 - Set a default HTTL via an engine configuration.
 - Switch off the HTTL check via an engine configuration if history cleanup is not used.
 
 ## <MarkerGuideline.Info /> History time to live not configured
 
 ![History time to live not configured](./img/history-time-to-live/info.png)
 
+In the screenshot above, note that the time to live must be defined under **History cleanup** in the properties panel.
```

Comment:
For accessibility purposes and to explain the screenshot, I added the following sentence. WDYT @toco-cam ?

Reply:
@christinaausley It is unfortunately not that easy, as there are three options to set the HTTL which are valid (Listed above the pic):
- Define HTTL per model directly in Desktop Modeler.
- Set a default HTTL via an engine configuration.
- Switch off the HTTL check via an engine configuration if history cleanup is not used.

Reply:
Who added the screenshot? The screenshot and your text only make sense for this bullet point "Define HTTL per model directly in Desktop Modeler."

Reply:
Hi @toco-cam -- this is actually based on [your PR](https://github.com/camunda/camunda-docs/pull/3324), if you'd like to make additional changes.

Reply:
Hi @christinaausley, ok, now it makes sense to me - all modeling guidance does have a screenshot displaying how the linting looks in the Modeler. 

None of the other guidance has a caption - I would recommend not having one here, too.